ci: add action to run Chromatic (#8)
ithaka/pharos: docs/development/testing.md

Because most Pharos components provide a visual aspect, it's important to test regressions to the visual appearance in addition to the interactive behavior.

> [Chromatic](https://www.chromatic.com/) is run on pull requests that include changes to the `core` package, the project's dependencies, or Storybook configuration. This will build an emphemeral Storybook instance, screenshot each story, and determine if any stories are different from the baseline.

Reviewer: ```suggestion
[Chromatic](https://www.chromatic.com/) runs against pull requests that include changes to the `core` package, the project's dependencies, or Storybook configuration. This will build an emphemeral Storybook instance, screenshot each story, and determine if any stories are different from the baseline.
```
